zephyr/samples/someip_server/src/main.cpp

```
@@ -0,0 +1,114 @@
+
+int main() {
+    printf("=== SOME/IP Server (Zephyr native_sim) ===\n");
+
+    Server server;
+    if (!server.start()) {
+        return 1;
+    }
+
+    printf("[server] Waiting for requests... (runs until killed)\n");
+
+    while (true) {
```

Cppcheck (2.19.0)

[style] The function 'set_session_id' is never used.

(unusedFunction)